src/docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -2125,6 +2151,18 @@ server.3=zoo3:2888:3888</programlisting>
             usage limit that would cause the system to swap.</para>
           </listitem>
         </varlistentry>
+
+        <varlistentry>
+          <term>Publicly accessible deployment</term>
+
+          <listitem>
+            <para>ZooKeeper was not designed to be opened to the outside world. A ZooKeeper
```

Comment:
Both of these sentences are a little bit redundant, perhaps this can be simplified to:
A ZooKeeper ensemble is expected to operate in a trusted computing environment. It is thus recommended to deploy ZooKeeper behind a firewall.